doc/source/whatsnew/v0.22.0.txt

```diff
@@ -26,6 +26,7 @@ Other Enhancements
 - :func:`pandas.tseries.frequencies.to_offset` now accepts leading '+' signs e.g. '+1h'. (:issue:`18171`)
 - :class:`pandas.io.formats.style.Styler` now has method ``hide_index()`` to determine whether the index will be rendered in ouptut (:issue:`14194`)
 - :class:`pandas.io.formats.style.Styler` now has method ``hide_columns()`` to determine whether columns will be hidden in output (:issue:`14194`)
+- :func:`Series.fillna` now accepts a Series or a dict as a ``value`` (:issue:`17033`)
```

Comment:
can you add another note in api breaking changes. note that the previous exception was a ``ValueError``, now its a ``TypeError`` (good change).  use this PR number for that note.